diff --git a/mothership/template.php b/mothership/template.php
index 8ff0723..a699b3a 100644
--- a/mothership/template.php
+++ b/mothership/template.php
@@ -63,6 +63,19 @@ function mothership_preprocess(&$vars, $hook) {
     Go through all the hooks of drupal and give em epic love
   */
 
+  // Things common to both html & maintenance page templates.
+  if ($hook == 'html' || $hook == 'maintenance_page') {
+    // Selectivizr.
+    $vars['selectivizr'] = '';
+    if (theme_get_setting('mothership_selectivizr')) {
+      $vars['selectivizr'] .= '<!--[if (gte IE 6)&(lte IE 8)]>' . "\n";
+      $vars['selectivizr'] .= '<script type="text/javascript" src="http://cdnjs.cloudflare.com/ajax/libs/selectivizr/1.0.2/selectivizr-min.js"></script>' . "\n";
+      $vars['selectivizr'] .= '<![endif]-->' . "\n";
+    }
+
+    $vars['appletouchicon'] = $appletouchicon;
+  }
+
   if ( $hook == "html" ) {
     // =======================================| HTML |========================================
 
@@ -139,14 +152,6 @@ function mothership_preprocess(&$vars, $hook) {
       drupal_add_js('http://cdnjs.cloudflare.com/ajax/libs/modernizr/2.0.6/modernizr.min.js', 'external');
     }
 
-    //---- selectivizr
-    $vars['selectivizr'] = '';
-    if(theme_get_setting('mothership_selectivizr')) {
-      $vars['selectivizr'] .= '<!--[if (gte IE 6)&(lte IE 8)]>' . "\n";;
-      $vars['selectivizr'] .= '<script type="text/javascript" src="http://cdnjs.cloudflare.com/ajax/libs/selectivizr/1.0.2/selectivizr-min.js"></script>' . "\n";;
-      $vars['selectivizr'] .= '<![endif]-->' . "\n";;
-    }
-
     //---html5 fix
     $vars['html5iefix'] = '';
     if(theme_get_setting('mothership_html5')) {
@@ -155,8 +160,6 @@ function mothership_preprocess(&$vars, $hook) {
       $vars['html5iefix'] .= '<![endif]-->';
     }
 
-    $vars['appletouchicon'] = $appletouchicon;
-
     //-----<body> CSS CLASSES  -----------------------------------------------------------------------------------------------
     //Remove & add cleasses body
 
@@ -516,8 +519,6 @@ function mothership_preprocess(&$vars, $hook) {
     // =======================================| maintenance page |========================================
 
     $vars['path'] = $path;
-    $vars['appletouchicon'] = $appletouchicon;
-    $vars['selectivizr'] = $selectivizr;
     $vars['theme_hook_suggestions'][] = 'static__maintenance';
 
   }
